src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1294,7 +1294,7 @@ def _construct_app_and_env_for_source_or_repo(self):
         image = None if self.get_argument_image().__eq__(HELLO_WORLD_IMAGE) else _reformat_image(self.get_argument_source(), self.get_argument_repo(), self.get_argument_image())
```

Comment:
Hi @snehapar9 
Please add a description for this change in history.rst.

Reply:
Could you please rerun tests in `test_containerapp_create_update_with_source.py` and update the recording file?

Reply:
Updated.